src/main/resources/static/js/MainWindow.js

```diff
@@ -383,10 +383,10 @@ function saveConfig() {
 	var map = mappingData;
 
 	var cfgName = prompt("Name of Configuration:");
-	if (cfgName === null) {
+	if (cfgName == null) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4872216)